build_locally.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/sh
 
+# Install sphinx-build if it isn't there yet
+if ! type "sphinx-build" > /dev/null
```

Comment:
I tried to use the same logic as [moveit_tutorials](https://github.com/ros-planning/moveit_tutorials/blob/master/build_locally.sh#L4) but the script would always attempt to reinstall sphinx.

`if ! command -v sphinx-build &> /dev/null`